src/ts/components/seekbar.ts

```diff
@@ -161,16 +161,14 @@ export class SeekBar extends Component<SeekBarConfig> {
         if (player.getMaxTimeShift() === 0) {
           // This case must be explicitly handled to avoid division by zero
           this.setPlaybackPosition(100);
-        }
-        else {
+        } else {
```

Comment:
unnecessary change

Reply:
these changes were mainly to refine the code, might be non-relevant for this PR but it's ok to keep consistency I guess